common-design/demo.html

```diff
@@ -1,29 +1,18 @@
 ---
+options:
+- hide_header_footer
+- include_cd
+- include_cd_header
+- include_cd_footer_soft
+- include_cd_footer
```

Comment:
@left23 perhaps the best demonstration of what really happened here is the `demo.html` file which used to be a hardcoded collection of exceptions. Now it's running off the main layout and `<head>` markup with a series of options to change what assets/markup are loaded.

Reply:
Yes. This seems much nicer. And maybe gets us closer to including variants without having to duplicate loads of markup and files.